tests/HtmlTest.php

```diff
@@ -1,416 +1 @@
      $menu = Html::getMenuInfos();
      $this->assertEquals(8, count($menu));

      $expected = [
         'assets',
         'helpdesk',
         'management',
         'tools',
         'plugins',
         'admin',
         'config',
         'preference'
      ];
      $this->assertEquals($expected, array_keys($menu));

      $expected = [
         'Computer',
         'Monitor',
         'Software',
         'NetworkEquipment',
         'Peripheral',
         'Printer',
         'CartridgeItem',
         'ConsumableItem',
         'Phone'
      ];
      $this->assertEquals('Assets', $menu['assets']['title']);
      $this->assertEquals($expected, $menu['assets']['types']);

      $expected = [
         'Ticket',
         'Problem',
         'Change',
         'Planning',
         'Stat',
         'TicketRecurrent'
      ];
      $this->assertEquals('Assistance', $menu['helpdesk']['title']);
      $this->assertEquals($expected, $menu['helpdesk']['types']);

      $expected = [
         'SoftwareLicense',
         'Budget',
         'Supplier',
         'Contact',
         'Contract',
         'Document'
      ];
      $this->assertEquals('Management', $menu['management']['title']);
      $this->assertEquals($expected, $menu['management']['types']);

      $expected = [
         'Project',
         'Reminder',
         'RSSFeed',
         'KnowbaseItem',
         'ReservationItem',
         'Report',
         'MigrationCleaner'
      ];
      $this->assertEquals('Tools', $menu['tools']['title']);
      $this->assertEquals($expected, $menu['tools']['types']);

      $expected = [];
      $this->assertEquals('Plugins', $menu['plugins']['title']);
      $this->assertEquals($expected, $menu['plugins']['types']);

      $expected = [
         'User',
         'Group',
         'Entity',
         'Rule',
         'Profile',
         'QueuedMail',
         'Backup',
         'Event'
      ];
      $this->assertEquals('Administration', $menu['admin']['title']);
      $this->assertEquals($expected, $menu['admin']['types']);

      $expected = [
         'CommonDropdown',
         'CommonDevice',
         'Notification',
         'SLA',
         'Config',
         'Control',
         'Crontask',
         'Auth',
         'MailCollector',
         'Link',
         'Plugin'
      ];
      $this->assertEquals('Setup', $menu['config']['title']);
      $this->assertEquals($expected, $menu['config']['types']);

      $this->assertEquals('My settings', $menu['preference']['title']);
      $this->assertNull($menu['preference']['types']);
      $this->assertEquals('/front/preference.php', $menu['preference']['default']);

   }

   /**
    * @covers Html::getCopyrightMessage()
    */
   public function testGetCopyrightMessage() {
      $message = Html::getCopyrightMessage();
      $this->assertContains(GLPI_VERSION, $message, 'Invalid GLPI version!');
      $this->assertContains(date('Y'), $message, 'Invalid copyright date!');
   }
}

```

Comment:
I mean the file has been changed in this commit... No doubt it is a mistake, but no doubt this is in the commit :)